### DISCUSSION TITLE
No Snowpark Container Services resource?

### DISCUSSION THREAD

**Discussion by tblake84** [Snowflake-Labs/terraform-provider-snowflake#3114] [General]
Is there any plan to integrate SPCS as an available resource in this provider?  What is the recommended approach to deploying an SPCS service using terraform in the interim?

**Comment by sfc-gh-jcieslak** [comment]
Hey @tblake84 👋 
Currently, we are focusing on the objects from the following lists: 
- https://github.com/Snowflake-Labs/terraform-provider-snowflake/blob/main/v1-preparations/ESSENTIAL_GA_OBJECTS.MD
- https://github.com/Snowflake-Labs/terraform-provider-snowflake/blob/main/v1-preparations/REMAINING_GA_OBJECTS.MD

As you can see, it's a lot, so it will take some time until we get to the objects outside of those lists. SPCS is on our list in the backlog, but because there are a lot of topics to be done beforehand, I cannot give you an ETA for that. If you (or anyone from your team) would like to contribute, we are open to prs, but we expect to be familiar with [contribution guidelines](https://github.com/Snowflake-Labs/terraform-provider-snowflake/blob/main/CONTRIBUTING.md) and to contact us before creating a pr (e.g. through this discussion or newly created issue) so we could answer your questions or give some guidelines on what are our expectations. The only workaround, for now, is to use the [unsafe_execute](https://registry.terraform.io/providers/Snowflake-Labs/snowflake/latest/docs/resources/unsafe_execute) resource.
